### ISSUE TITLE
Incremental build slowdown

### ISSUE THREAD

**Issue by nasushkov** [halt-hammerzeit/webpack-isomorphic-tools#135]
Hi
I encounter a significant slowdown in the incremental build performance after adding webpack-insomorphic-tools. In particular, it adds around 2,5-3,5 sec to the existing 3,5 sec.  Here is my isomorphic config: 

```
 module.exports = {
  webpack_assets_file_path: path.join(publicPath(), 'webpack-assets.json'),
  patch_require: true,
  debug: true,
  alias: resolve.alias,
  assets: {
    images: {
      extensions: [
        'jpeg', 'jpg', 'png', 'gif', 'svg'
      ],
    },
    fonts: {
      extensions: [
        'woff', 'ttf', 'woff2', 'eot',
      ],
    },
    style_modules: {
      extensions: ['scss', 'css'],
    },
  },
};
```
I also have many errors in webpack-isomorphic-tools/plugin output: 

> [webpack-isomorphic-tools/plugin] [error] Error: More than a single candidate module was found in webpack stats for require()d path "!!../../../node_modules/css-loader/index.js!../../../node_modules/postcss-loader/index.js!../../../node_modules/sass-loader/index.js?outputStyle=expanded!./index.scss". Enable "debug: true" flag in webpack-isomorphic-tools configuration for more info.

As I understand, it can be related to the fact that ,/index.scss is used as a name for many .scss files in this App.  Can it be the reason for a slowdown or there may be something else?

Thanks

**Comment by halt-hammerzeit** [comment]
Try using `universal-webpack` instead.
The "More than a single candidate module" issue is not solvable afaik

**Comment by nasushkov** [comment]
Well ok, but it can be a significant effort for now as we already implemented everything with webpack-isomorphic-tools and we're still using Webpack 1 for this project. Do you know any other reasons for a slowdown, so we can investigate?

**Comment by halt-hammerzeit** [comment]
Even I don't know what's happening deep down there

**Comment by nasushkov** [comment]
@halt-hammerzeit anyway thanks for a fast response

**Comment by halt-hammerzeit** [comment]
By the way, if you have `3.0.0` or `3.0.1` version installed it had an unrelated bug I fixed today so upgrade to `3.0.2`.
If you're on `2.x` then there's no need to upgrade.
